Muehle_Logic.ipynb

```diff
@@ -410,14 +410,17 @@
     "                update_board(game)\n",
     "            else:\n",
     "                raise InvalidMoveException()\n",
+    "                change_status_label('Du kannst Deinen Stein nur auf ein leeres Feld setzen. Bitte probiere es erneut!')\n",
     "                print('Du kannst Deinen Stein nur auf ein leeres Feld setzen. Bitte probiere es erneut!')\n",
     "        elif phase == 2 or phase == 3:\n",
     "            if game.move_or_jump == False: # -> Stein zum ziehen auswählen\n",
+    "                change_status_label('Zielort für den eigenen Stein auswählen.') # Statusausgabe\n",
```

Comment:
Das muss glaube ich nach der if-Abfrage kommen.

Reply:
Es wird bei mir angezeigt, wenn ich in Phase 3 einen Stein ausgewählt habe. Müsste also stimmen.

Reply:
Ja, aber es wird auch angezeigt, wenn man auf einen Stein des Gegners klickt.